Hestia/app/src/main/java/hestia/UI/HestiaApplication.java

```diff
@@ -9,4 +15,25 @@
 
 public class HestiaApplication extends Application {
 
+    private NetworkHandler networkHandler;
+
+    private static final String HESTIA_IP = "HESTIA.IP";
+    private static final String SERVER_IP = "IP_OF_SERVER";
+
+    @Override
+    public void onCreate() {
+        setupNetworkHandler();
+        super.onCreate();
+    }
+
+    private void setupNetworkHandler() {
+        SharedPreferences prefs = getSharedPreferences(HESTIA_IP, Context.MODE_PRIVATE);
+        String ip = prefs.getString(SERVER_IP, getApplicationContext().getString(R.string.default_ip));
+        networkHandler = new NetworkHandler(ip
+                , Integer.valueOf(getApplicationContext().getString(R.string.default_port)));
```

Comment:
I believe you can simply use R.string.default_port, which will return an int. I think I used it somewhere and it worked fine.